internal/aws/containerinsight/const.go

```diff
@@ -0,0 +1,212 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package containerinsight
+
+import (
+	"strings"
+	"time"
+)
+
+//define metric names, attribute names, metric types, and units for both EKS and ECS Container Insights
```

Comment:
Let's add space, `// ` throughout the comments I think it's more conventional.

Reply:
I will follow this convention in later PRs. Thanks for reviewing it!